.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,45 @@
+name: e2e
+on:
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
+  schedule:
+  - cron: '0 0 * * *' # daily to pick up releases
+jobs:
+  kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
I'd downgrade this to the v1 version as there's some problems with v2 checking out the wrong commits.

Reply:
Think we should downgrade the other actions as well?

Reply:
Yeah, that would be my preference, but let's do that in a separate PR.